tests/modules/test_json/test_decoding.py

```diff
@@ -122,9 +125,9 @@ class LoadTests(ModuleFunctionTestCase, TranspileTestCase):
     not_implemented_versions = {
     }
 
-    not_implemented = [
-        'test_fail',
-    ]
+    not_implemented = {
+        'test_fail'
```

Comment:
A minor thing worth flagging as "future protection" - once [this PR](https://github.com/python/black/pull/826) is merged and released, it's quite likely I'll be convinced to turn on Black. That PR means that the trailing comma on a non-tuple is considered a significant indicator that the content is a "list of items", and shouldn't be collapsed into a single line.

Reply:
This was the result of a merge. I'll revert it.